lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -57,7 +57,7 @@ def build_service_ui
 
         Dir.chdir(ui_service_dir) do
           shell_cmd("yarn set version 1.22.18") if RUBY_PLATFORM.include?("s390x")
-          shell_cmd("yarn install")
+          shell_cmd("yarn install --frozen-lockfile") # Switch --frozen-lockfile to --immutable once s390x is off of yarn 1.
```

Comment:
Nice.  I like the commend about s390x, because that's easily searchable :+1:

Reply:
Note, this might not be the right place for ui-classic (cause this is the ui-service build code).  The ui-classic code is here: https://github.com/ManageIQ/manageiq-ui-classic/blob/4cc3be00735dafeec3783258ed40502dcbf365ca/lib/tasks/manageiq/ui_tasks.rake#L12